solarforecastarbiter/reports/templates/pdf/macros.j2

```diff
@@ -125,11 +125,23 @@
     %- endif
     \VAR{pfxobs.original.data_object.interval_label} &
     \VAR{pfxobs.original.data_object.interval_length.total_seconds()//60 | int} min &
+    %- if pfxobs.original.forecast.constant_values is defined
```

Comment:
macro?

Reply:
Oh I guess i don't know about macro within macro

Reply:
I looked at it for a bit but had a hard time with it